source/common/http/async_client_impl.h

```diff
@@ -288,6 +288,7 @@ class AsyncStreamImpl : public AsyncClient::Stream,
     const Router::DirectResponseEntry* directResponseEntry() const override { return nullptr; }
     const Router::RouteEntry* routeEntry() const override { return &route_entry_; }
     const Router::Decorator* decorator() const override { return nullptr; }
+    const Router::RouteTracing* tracingConfig() const override { return nullptr; }
```

Comment:
Can you add a test for this once https://github.com/envoyproxy/envoy/pull/7042 merges.

Reply:
ok wlil take a look.